### PR TITLE
Italian SEO chapter translation

### PR DESCRIPTION
**Pull request by chefleo**
Makes progress on #585

### CODE REVIEW

**Review by bazzadp — APPROVED** [review]
LGTM. A few small tweaks

File: src/content/it/2020/seo.md [hunk 1]
```diff
@@ -0,0 +1,1219 @@
...
+fellowhuman1101_bio: 100% umano e ovviamente non un robot, Jamie Indigo districa le tecnologie per aiutare gli esseri umani ad accedere a informazioni utili e le aziende forniscono preziose esperienze digitali. Ha fondato <a hreflang="en" href="https://not-a-robot.com">Not a Robot</a> per consultarsi con precisione sugli aspetti umani della SEO tecnica, tra cui l'etica e l'inclusione nella tecnologia e nel settore della ricerca. Può scoprire che sta imparando in pubblico su <a href="https://twitter.com/Jammer_Volts">Twitter</a>.
+discuss: 2043
+results: https://docs.google.com/spreadsheets/d/1ram47FshAjzvbQVJbAQPgxZN7PPOPCKIK67VJZCo92c/
+queries: 07_SEO
```
Comment (bazzadp) [inline src/content/it/2020/seo.md:15]:
```suggestion
```

File: src/content/it/2020/seo.md [hunk 2]
```diff
@@ -0,0 +1,1219 @@
...
+<link rel="alternate" media="only screen and (max-width: 640px)">
+```
+
+Si consiglia ai siti web desktop che hanno versioni mobile separate di collegarsi ad essi utilizzando questo tag nella `head` del loro HTML. È stato rilevato che solo lo 0.64% delle pagine desktop analizzate includeva il tag con il valore dell'attributo `media` specificato.
+
+### Prestazioni web
+
+Avere un sito Web veloce è fondamentale per fornire un'ottima esperienza di ricerca utente. Per la sua importanza è da anni considerato come fattore di ranking dai motori di ricerca. Google ha inizialmente annunciato di utilizzare la velocità del sito come [fattore di ranking nel 2010](https://webmasters.googleblog.com/2010/04/using-site-speed-in-web-search-ranking.html), quindi [nel 2018 ha fatto lo stesso per le ricerche su dispositivi mobile](https://webmasters.googleblog.com/2018/01/using-page-speed-in-mobile-search.html).
+
+{# NOTE(authors): I've made some ruthless edits to this section to remove everything related to synthetic measurement of CWV, including the entire Lighthouse discussion, which is orthogonal to the real-user aspect of CWV. Please push back if you disagree with any of these edits. #}
```
Comment (bazzadp) [inline src/content/it/2020/seo.md:773]:
```suggestion
```

**Review by bazzadp — COMMENTED** [review]
More fixes

**Review by bazzadp — APPROVED** [review]

**Comment by bazzadp** [comment]
Thanks @chefleo !!

File: src/content/it/2020/seo.md [hunk 3]
```diff
@@ -0,0 +1,1217 @@
...
+
+Durante il controllo dell'utilizzo di AMP come versione alternativa di una pagina non basata su AMP, abbiamo scoperto che lo 0.69% delle pagine web mobile e lo 0.81% di quelle desktop includevano un tag `amphtml` che puntava a una versione AMP. Sebbene l'adozione sia ancora molto bassa, si tratta di un leggero miglioramento rispetto [ai risultati dello scorso anno](../2019/seo#amp), in cui solo lo 0.62% delle pagine mobile conteneva un link a una versione AMP.
+
+D'altra parte, nel valutare l'utilizzo di AMP come framework per lo sviluppo di siti Web, abbiamo riscontrato che solo lo 0.18% delle pagine mobile e lo 0.07% di quelle desktop specificava `<html amp>` o l'attributo emoji `<html ⚡>`, il quale vengono utilizzati per indicare le pagine basate su AMP.
+
+### Applicazioni Single-page
```
Comment (Zizzamia) [inline src/content/it/2020/seo.md:1172]: @chefleo I wonder if in Italian say Applicatione Single-page or we should keep it in english as Single Page Application. @giopunt thoughts? We might want even ask Twitter for this 😆

**Pull request merged**